### PR TITLE
Update browser extensions section so users are now aware about the dependency on the desktop application

### PR DESCRIPTION
**Pull request by Oxalin**
The new version 3 of the extension is now requiring the Buttercup Desktop application be installed and running. This a change in response to the upcoming Manifest V3 adopted by Google Chrome.

While the extension is now stating this requirement in the README file and in its notification tab, the user was left unaware of this requirement when downloading the extension.

Fix some warnings seen while testing the change.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
   "devDependencies": {
     "bulma": "^0.9.4",
     "es6-promise": "^4.2.8",
+    "npx": "^10.2.2",
```
Comment (perry-mitchell) [inline package.json]: Not needed, it's included with Npm

File: package.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "buttercup-website",
-  "version": "4.2.0",
+  "version": "4.2.1",
   "description": "Buttercup's Website",
   "scripts": {
-    "build": "next build && next export",
+    "build": "next build",
```
Comment (perry-mitchell) [inline package.json]: This is needed for release, please don't change it :)

File: package.json [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
     "deploy": "npm run build && aws s3 sync --acl public-read ./out/ s3://buttercup.pw/",
     "dev": "next",
     "format": "prettier --write '{components,pages,utils}/**/*.js'",
-    "start": "next start",
+    "start": "npx serve@latest out",
```
Comment (perry-mitchell) [inline package.json]: Same here

File: package.json [hunk 4]
```diff
@@ -1,13 +1,13 @@
 {
   "name": "buttercup-website",
-  "version": "4.2.0",
+  "version": "4.2.1",
```
Comment (perry-mitchell) [inline package.json]: This is handed during please. Please revert it.

Also, you might revert the changes to package-lock as it has a ton of changed lines.

**Review by perry-mitchell — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Oxalin) [reply package.json]: ok. It was not installed when I first tried it.

File: package.json (hunk 2 above)
Comment (Oxalin) [reply package.json]: There was a warning stating that it was _now_ unneeded (may have been needed previously) when "output: export" was present in next.config.js

I can remove it, but is it really still needed?

File: package.json (hunk 4 above)
Comment (Oxalin) [reply package.json]: Ah, I wasn't sure how it was handled. OK

**Comment by Oxalin** [comment]
I'll send a new PR with just the update to the extension section.

**Comment by perry-mitchell** [comment]
Thanks for that. Yeah it probably all needs an update (the install / release process) but for now it works automatically.
